app/models/client.rb

```diff
@@ -1,9 +1,9 @@
 class Client < ApplicationRecord
-  has_many :orders
+  has_many :orders, dependent: :destroy
```

Comment:
Nope -- that's dangerous
I don't think we don't want deleted clients to delete related orders